### PR TITLE
greening - added creation user, added smoke tag

### PR DESCRIPTION
**Pull request by TetianaParanich**

### CODE REVIEW

File: cypress/integration/data-import/mrc-import-with-matching-001.spec.js [hunk 1]
```diff
@@ -17,25 +17,30 @@ import FileManager from '../../support/utils/fileManager';
...
-  beforeEach(() => {
-    cy.login(
-      Cypress.env('diku_login'),
-      Cypress.env('diku_password')
-    );
-    cy.getAdminToken();
-
-    cy.visit(TopMenu.dataImportPath);
-
+  let user = {};
```
Comment (dmitriyfolio) [inline cypress/integration/data-import/mrc-import-with-matching-001.spec.js:23]: fix linter issue

Comment (TetianaParanich) [reply cypress/integration/data-import/mrc-import-with-matching-001.spec.js:23]: Done

**Review by dmitriyfolio — APPROVED** [review]

File: cypress/support/fragments/inventory/InventoryInstanceEdit.js [hunk 2]
```diff
@@ -81,12 +81,15 @@ export default {
...
-    cy.do([
-      Button('Add identifier').click(),
-      Accordion('Identifier').find(Select({ name:'identifiers[0].identifierTypeId' })).choose(identifier),
-      TextField({ name:'identifiers[0].value' }).fillIn(value),
-      saveAndCloseButton.click()]);
+    cy.expect(Accordion('Identifier').exists());
+    cy.do(Button('Add identifier').click());
+    cy.expect(Select('Type*').exists());
+    cy.expect(TextField('Identifier*').exists());
+    cy.do(Accordion('Identifier').find(Select({ name:'identifiers[0].identifierTypeId' })).choose(identifier));
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/inventory/InventoryInstanceEdit.js:88]: I suggest Accordion('Identifier') and other repeating elements move to const

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/support/fragments/inventory/InventoryInstanceEdit.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/support/fragments/inventory/InventoryInstanceEdit.js]: Done

**Pull request merged**